kolibri_explore_plugin/assets/src/app.js

```diff
@@ -62,6 +63,8 @@ class ExploreModule extends KolibriApp {
         false
       );
     }
+
+    window.kolibri = kolibriApi;
```

Comment:
I think we should add a build option or a configuration flag so the plugin can use this `window.kolibri` or the `window.kolibri` added by hashi. Without that configuration, we won't be able to continue testing this `api-migration` branch with Kolibri `release-v0.15.x` branch.